test/test-cases/functional/saic/test_vm_to_vm_commn_acl_outbound.py

```diff
@@ -0,0 +1,183 @@
+import json
+from pathlib import Path
+from pprint import pprint
+import time
+import pytest
+import sys
+sys.path.append("../utils")
```

Comment:
Looks like there are 2 folders for keeping utils. The second one is `test/test-cases/scale/saic)/dash_helper`.
We need to find a single place for all of them.

Reply:
I think utils folder would be better as we can utilize it for functional as well as scale scripts.. Further we can also discuss and identify the correct one.

Reply:
made utils folder and modified existing script as well